app/pages/home/homePageController.ts

```diff
@@ -25,20 +25,12 @@ export class HomePageController {
   };
 
   public update() {
-    this.Catalog.getCatalogItems().then( (catalogServiceItems: any) => {
+    this.Catalog.getCatalogItems(false).then( (catalogServiceItems: any) => {
```

Comment:
I do not see where 'this.Catalog.getCatalogItems(true)' is ever called, I assume it's done in the console?